README.md

```diff
@@ -161,6 +161,14 @@ Reports usage of superfluous prefix or suffix "Interface" for interfaces.
 
 Reports usage of superfluous suffix "Exception" for exceptions.
 
+#### SlevomatCodingStandard.Classes.DisallowOneLinePropertyDocComment 🔧
+
+Require comments with one-line content to be written as multi-liners.
+
+#### SlevomatCodingStandard.Classes.RequireOneLinePropertyDocComment 🔧
+
+Require comments with one-line content to be written as one-liners.
```

Comment:
Typo -> "with multi-line content" :)

Reply:
Actually, no, but changing to _single-line_.

Reply:
Oh, I understand now.